src/icons/dialog.svg

```diff
@@ -0,0 +1 @@
+<svg fill="none" viewBox="0 0 32 33" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><filter id="a" color-interpolation-filters="sRGB" filterUnits="userSpaceOnUse" height="32" width="32" x="0" y=".587891"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" result="hardAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="2"/><feGaussianBlur stdDeviation="2"/><feColorMatrix type="matrix" values="0 0 0 0 0.376471 0 0 0 0 0.380392 0 0 0 0 0.439216 0 0 0 0.16 0"/><feBlend in2="BackgroundImageFix" mode="normal" result="effect1_dropShadow_716_7708"/><feColorMatrix in="SourceAlpha" result="hardAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset/><feGaussianBlur stdDeviation=".5"/><feColorMatrix type="matrix" values="0 0 0 0 0.156863 0 0 0 0 0.160784 0 0 0 0 0.239216 0 0 0 0.04 0"/><feBlend in2="effect1_dropShadow_716_7708" mode="normal" result="effect2_dropShadow_716_7708"/><feBlend in="SourceGraphic" in2="effect2_dropShadow_716_7708" mode="normal" result="shape"/></filter><g filter="url(#a)"><path clip-rule="evenodd" d="m4 10.5879c0-4.41829 3.58172-8.00001 8-8.00001h8c4.4183 0 8 3.58172 8 8.00001v8c0 4.4183-3.5817 8-8 8h-8c-4.41828 0-8-3.5817-8-8z" fill="#fff" fill-rule="evenodd"/></g></svg>
```

Comment:
Is this logo being rendered? Not seeing it in the screenshot

Reply:
the logo is a white square as it shows^

Reply:
Ohh sorry I thought it was supposed to be a university 😅

Reply:
removed this placehoder